checker/tests/test_AnDO_Engine.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+
+import AnDO_engine as andoE
+import json
+import os
+path=os.getcwd()
+
+
+class test_AnDO(unittest.TestCase):
+
+    
+    def setUp(self):
+        pass
+    
+    def test_AnDO_Func_level_1(self):
```

Comment:
please add a short documentation explaining what each function is testing exactly...

also, you use "level_1", "level_2" etc. in other places, you use "subject", "session" etc. (function is_subject, class SubjectError); you should be consistent in all the code!

Reply:
do you want to do this now, or after the rebase? if after, I think you should at least put something in the code to remember to do it ;)

Reply:
I think we can do it now become it's not related to the new specs .

Reply:
ok, go for it! ;)

Reply:
ok, go for it!